build-tools/packages/build-cli/docs/bump.md

```diff
@@ -144,3 +145,25 @@ EXAMPLES
 
     $ flub bump deps server -t latest
 ```
+
+## `flub bump version`
```

Comment:
I feel like "version" is a bit generic of a name for this type of command. What do you think about something more specific like "generateManifest"?